src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -173,11 +173,6 @@ export const gravityStitchingEnvironment = (
         curatedMarketingCollections(size: Int): [MarketingCollection]
       }
 
-      extend type System {
-        algolia: Algolia
-      }
```

Comment:
I'm assuming we are fine removing this field outright (as opposed to marking it a deprecated)

Reply:
[Wouldn't this](https://github.com/artsy/eigen/blob/a3871266daf9d7d836bc62fb8656195210e22ffb/src/app/Scenes/Search/Search.tsx#L247) break old versions of the app?

Reply:
Because the Algolia contract ends this week, we can expect those API calls to start falling any moment now too. 

So search functionality in old versions of the app will _definitely_ be broken whether we do this or not, the only question is to what degree this will be graceful. 

I think we confirmed earlier that old versions would just show a blank screen and not an error once Algolia API access is cut off — this change maybe makes it more likely that there will be a hard error screen. 

But, at this point, 4 months out from the cutover, we're talking [essentially 0% of users (iOS)](https://artsy.looker.com/explore/frontend_analytics/eigen_screens?qid=7KaBqF2IikzeaTSuXIYjPa&origin_space=712&toggle=vis) on versions prior to 8.9.0. (I'm not sure how to find the equivalent dash for Android, I'll try to locate...)

I'd favor pruning the schema of (nearly) unused fields, esp if they can't actually return any data. But I'm open to argument on this.

Reply:
That plan sounds good to me @anandaroop. Keeping the schema compatible for a bit longer would retain our expected degraded state for users still on old versions, which is basically that the Top and Artworks pills would work normally but all other pills would be greyed out and inaccessible.

We haven't tested the old app experience with an incompatible schema. It's possible that that would cause a more intrusive error state.

Reply:
@dblandin true, what we tested back then was if algolia was failing silently when metaphysics either returned an empty string or weird invalid key for this.

Also happy to help test out any weird edge cases.